crates/core/src/fnc/crypto.rs

```diff
@@ -10,6 +10,10 @@ pub fn blake3((arg,): (String,)) -> Result<Value, Error> {
 	Ok(blake3::hash(arg.as_bytes()).to_string().into())
 }
 
+pub fn joaat((arg,): (String,)) -> Result<Value, Error> {
+	Ok(joaat::hash_ascii_lowercase(arg.as_bytes()).to_string().into())
+}
```

Comment:
suggestion: better stick with a number output. We can then later convert it to a `string`, if needed.